src/interpreter/bytecode-array-builder.cc

```diff
@@ -1394,13 +1394,28 @@ BytecodeArrayBuilder& BytecodeArrayBuilder::RecordReplayAssertValue(const std::s
   return *this;
 }
 
-BytecodeArrayBuilder& BytecodeArrayBuilder::RecordReplayInstrumentation(const char* kind,
-                                                                        int source_position) {
+int BytecodeArrayBuilder::RecordReplayRegisterInstrumentationSite(
+    const char* kind, int source_position) {
+  if (!strcmp(kind, "breakpoint") && source_position != kNoSourcePosition &&
```

Comment:
This strcmp call is ugly, it would be nice if there were two functions where only one of them checks for duplicates but this is fine I guess.